README.md

```diff
@@ -2,23 +2,91 @@
+`cri-containerd` is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
 ![cri-containerd](./docs/cri-containerd.png)
 ## Current Status
-We are currently working toward an alpha version to be used in conjunction with Kubernetes 1.7. See the [roadmap](./docs/proposal.md#roadmap-and-milestones) for information about current and future milestones.
+`cri-containerd` is in alpha. This release is for use with Kubernetes 1.7. See
+the [roadmap](./docs/proposal.md#roadmap-and-milestones)
+for information about current and future milestones.
+## Getting started
+### Binary Dependencies and Specifications
+The current release of cri-containerd has been tested with:
+* [containerd](https://github.com/containerd/containerd) v0.2.3-1098-g8ed1e24
+* [runc](https://github.com/opencontainers/runc) v1.0.0-rc3
+* [CNI](https://github.com/containernetworking/cni) v0.4.0
+
+As containerd and runc move to their respective general availability releases,
+we will do our best to rebase/retest `cri-containerd` with these releases on a
+weekly/monthly basis. Similarly, given that `cri-containerd` uses the Open
+Container Initiative (OCI) [image](https://github.com/opencontainers/image-spec)
+and [runtime](https://github.com/opencontainers/runtime-spec) specifications, we
+will also do our best to update `cri-containerd` to the latest releases of these
+specifications as appropriate.
+### Install Dependencies
+1. runc dependencies:
+* Install the libsecomp development library appropriate for your distribution,
+`libseccomp-dev`(Ubuntu, Debian) / `libseccomp-devel`(Fedora, CentOS, RHEL). On
+releases of Ubuntu <=Trusty and Debian <=jessie a backport version of
+`libsecomp-dev` is required. See [travis yaml](.travis.yml) for an example on
+trusty.
+2. containerd dependencies:
+* btrfs-tools(Ubuntu, Debian) / btrfs-progs-devel(Fedora, CentOS, RHEL)
+3. Install and setup a go1.8.x development environment.
+4. Make a local clone of this repository.
+5. Install binary dependencies by running the following command from your github.com/kubernetes-incubator/cri-containerd/ path:
+```shell
+# Note: install.deps installs the above mentioned runc, containerd, and CNI
+# binary dependencies as well as cri-tools (discussed below). install.deps is
+# only provided for general use and ease of testing. To customize `runc`
+# and `containerd` build tags and/or to configure `cni`, please follow
+# instructions in their documents.
+make install.deps
+```
+### Build and Install cri-containerd
+To build and install cri-containerd enter the following commands from your github.com/kubernetes-incubator/cri-containerd/ path:
+```shell
+make
+sudo make install
+```
+### Validate Your cri-containerd Setup
+Another Kubernetes incubator project called [cri-tools](https://github.com/kubernetes-incubator/cri-tools)
```

Comment:
1) CRI validation test includes the validation and benchmark.
2) Their document has already linked to cri-tools, and explained what is cri-tools. We really don't need to mention it at the beginning here.

```
[CRI validation test](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md) is a test suite validating that a Container Runtime Interface (CRI) implementation such as `cri-containerd` meets all the requirements.
```

If you want, you can then add:
```
It's part of [`cri-tools`](https://github.com/kubernetes-incubator/cri-tools) which contains a group of toolkits for CRI development.
```